### PR TITLE
Use v1beta1 networking apis instead of v1alpha3

### PR DESCRIPTION
**Pull request by djazayeri**
### Background 

We use the alpha versions of APIs, but we should use the promoted ones.

### Fixes 
N/A

### Change Summary
Use v1beta1 networking APIs instead of v1alpha3.

### Additional Notes
N/A

### Testing Procedure
Not tested. This should be a trivial change.

### Related PRs or Issues 
N/A

### CODE REVIEW

**Review by bourgeoisor — APPROVED** [review]
LGTM

**Comment by djazayeri** [comment]
@bourgeoisor  I can't merge now (though maybe because we're missing some kind of approver), so yes, please merge.

**Pull request merged**
